src/utils/isReactComponentClass.js

```diff
@@ -44,6 +44,26 @@ export default function isReactComponentClass(
     return true;
   }
 
+  // check for @extends React.Component in docblock
+  if (path.parentPath.value && path.parentPath.value) {
```

Comment:
Why the doubled-up check?

Reply:
Whoops, should've been `(path.parentPath&& path.parentPath.value)`